app/views/requirements/edit.html.erb

```diff
@@ -0,0 +1,9 @@
+<% content_for :sidebar do %>
+  <h2>Edit Requirement</h2>
+  <h3>(for <%= @requirement.task.title %>)</h3>
+  <%= sidebar_button_link 'Return to Task', @requirement.task %>
+  <%= sidebar_button_link 'Return to Requirements List', requirements_path %>
+  <%= sidebar_button_link 'Show Requirement', @requirement %>
```

Comment:
Please change this to `Cancel Edit`, which is what it does. It would be ideal if it returned from where they started.
I'll probably try to change this to `Cancel Edit` in all the places.